pyxem/utils/indexation_utils.py

```diff
@@ -40,6 +40,40 @@
     "phase_index rotation_matrix match_rate error_hkls total_error scale center_x center_y".split(),
 )
 
+def optimal_fft_size(target, real = False):
+    """Wrapper around scipy function next_fast_len() for calculating optimal FFT padding.
+    scipy.fft was only added in 1.4.0, so we fall back to scipy.fftpack
+    if it is not available. The main difference is that next_fast_len()
+    does not take a second argument in the older implementation.
+
+    Parameters
+    ----------
+    target : int
+        Length to start searching from. Must be a positive integer.
+    real : bool, optional
+        True if the FFT involves real input or output, only available
+        for scipy > 1.4.0
+    Returns
+    -------
+    int
+        Optimal FFT size.
+    """
+
+    try:
+        from scipy.fft import next_fast_len
+
+        support_real = True
+
+    except ImportError:
+        from scipy.fftpack import next_fast_len
+
+        support_real = False
+
+    if support_real:
```

Comment:
and also here `#pragma no cover` as you can't know which path the travis will be running through (and it should only run through one)